qadence/backends/pyqtorch/backend.py

```diff
@@ -56,7 +56,7 @@ def circuit(self, circuit: QuantumCircuit) -> ConvertedCircuit:
 
         abstract = transpile(*transpilations)(circuit)  # type: ignore[call-overload]
         ops = convert_block(abstract.block, n_qubits=circuit.n_qubits, config=self.config)
-        native = pyq.QuantumCircuit(abstract.n_qubits, ops)
+        native = pyq.QuantumCircuit(abstract.n_qubits, ops, diff_mode="adjoint")
```

Comment:
I guess this should be added to the Backend configuration, right? Do you think it makes sense to have adjoint as the default method?

Reply:
removed, was just a test. this MR wont contain the adjoint yet, just the new pyq